ci/__main__.py

```diff
@@ -10,6 +10,9 @@
     "numpy>=1.13.0"
 ]
 
+ADDITIONAL_PIP_DEPS = [
```

Comment:
Originally this PR may have needed to utilise the [mdtraj](http://mdtraj.org/1.9.3/) library for file handling (especially reading of `.xtc` binary files). This is a useful library that is available through `pip`, but currently unsupported by `edm`. At the moment we have been able to avoid this requirement, but the `pip` installation code may come in handy for future functionality.